composer.json

```diff
@@ -30,6 +30,7 @@
         "ext-filter": "*",
         "ext-json": "*",
         "container-interop/container-interop": "^1.2",
+        "laminas/laminas-diactoros": "^2.13.0",
```

Comment:
Overall, I think this does bring value, and is implemented the right/safe way, but it also leads to the added dependency, which is a lot :thinking: 

Not sure what to do here. /cc @weierophinney maybe?

Reply:
I would likely copy over that functionality here and refactor to work with laminas-http. Since Diactoros is a PSR-7 library, there's a mismatch with the primary target of laminas-view (laminas-mvc). There's a reason we provide alternate url() and serverUrl() helpers in the mezzio-template-laminasview package...

Reply:
Can we/I create an entirely new library to contain everything that's MVC specific so that it can be stripped out in 3.0 - this could include all the supporting code around MVC events. If that's an agreeable path, I could just abandon this patch and deprecate the helper here entirely instead?

Reply:
I think that would be perfect, @gsteel!

Reply:
And we have to make sure that there is also a solution to create URLs with the same helpers when using laminas-cli. The server URL and the base path must be able to set via configuration, factory or something else.

Reply:
@froschdesign I'd argue that should be via separate helpers. Not quite sure how to manage it (right now, CLI config is the same as web config), but having separate helpers for separate contexts makes a lot of sense here to me.

Reply:
@weierophinney 

I've made a start on this with just the ServerUrl and Url helpers specifically for MVC:

https://github.com/gsteel/laminas-mvc-view/pull/1

If this is a good enough starting point, is transferring the repo ownership an option?

Reply:
Yes - we usually vote on any new packages in the TSC; this sort of thing can happen between meetings, though. Feel free to initiate; contact me in Slack for tips on how to do it.

Looking forward to further decoupling!